NSS-Day-Cohort-64/rare-python-server-donkey-puppets
# Added do_Post for tags and create_tag function

Added tags to the do_Post method in request_handler and create_tag function to create a new tag.

To test:
- start debugger and go to postman and enter http://localhost:8088/tags for the url
- change the method to POST
- create a body to send in raw format 
{
    "label": "enter whatever here"
}

- click send and verify that the status is "201 created"
- then go to your sqlite database and verify that the new tag was created
